### ISSUE TITLE
reduce information provided about the project

### ISSUE THREAD

**Issue by JuliaLawall** [sagesharp/outreachy-django-wagtail#289]
Many applicants send email to find out information that is actually in the project description.  Maybe it can help a little to reduce the amount of information provided.  For example my project description contains:

Some projects accept contributions through a project repository. This project has not provided a link to a project repository.

Maybe these two sentences could be just dropped completely, rather than having one sentence that provides irrelevant information and then another sentence that says that it is not relevant.

**Comment by sagesharp** [comment]
I agree that the project pages could use an overhaul. It's difficult to write a template that applies to all projects. :)

Part of the reason the project page is so wordy is because it's trying to provide background to people who are new to open source. For example, they may not understand what a repository is or what IRC is.

What I think should be done instead is to have various "tip" dialog boxes that point newcomers to open source to a larger FAQ document. That will also help people who are trying to prep for the round. This should be part of #291.

**Comment by JuliaLawall** [comment]
On Thu, 4 Apr 2019, Sage Sharp wrote:

>
> I agree that the project pages could use an overhaul. It's difficult to
> write a template that applies to all projects. :)
>
> Part of the reason the project page is so wordy is because it's trying to
> provide background to people who are new to open source. For example, they
> may not understand what a repository is or what IRC is.
>
> What I think should be done instead is to have various "tip" dialog boxes
> that point newcomers to open source to a larger FAQ document. That will also
> help people who are trying to prep for the round. This should be part of
> #291.

It seems like it could be a good idea.  It would give people access to
that information when they need it, while not bombarding them with details
when they just want to find out what the project is about.

thanks,
julia
